integration-tests/ccip-tests/testconfig/README.md

```diff
@@ -423,6 +423,32 @@ Example usage:
 TTL = "11h"
 ```
 
+### CCIP.Env.Logging
+
+Specifies the logging configuration for the test. Imported from [LoggingConfig](https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/config/logging.go#L11) in chainlink-testing-framework.
```

Comment:
@b-gopalswami I don't think this is true. Could you please verify how this configuration should look like?